packages/gatsby-plugin-manifest/src/__tests__/gatsby-node.js

```diff
@@ -0,0 +1,44 @@
+jest.mock(`fs`, () => {
+  return {
+    existsSync: jest.fn().mockImplementation(() => true),
+    writeFileSync: jest.fn(),
+    statSync: jest.fn(),
+  }
+})
+const fs = require(`fs`)
+const { onPostBootstrap } = require(`../gatsby-node`)
+
+describe(`Test plugin manifest options`, () => {
+  it(`correctly works with default parameters`, async () => {
+    await onPostBootstrap([], {
+      name: `GatsbyJS`,
+      short_name: `GatsbyJS`,
+      start_url: `/`,
+      background_color: `#f7f0eb`,
+      theme_color: `#a2466c`,
+      display: `minimal-ui`,
+    })
+    expect(fs.writeFileSync.mock.calls).toMatchSnapshot()
```

Comment:
can you change this to something like:
```js
    const [filePath, contents] = fs.writeFileSync.writeFile.mock.calls[0]
    expect(filePath).toEqual(path.join(`public`, `manifest.webmanifest`))
    expect(contents).toMatchSnapshot()
```
this test will fail on windows because of different path separator saved in snapshot

Reply:
done! sorry forgot windows.